src/services/Safeplace.ts

```diff
@@ -0,0 +1,30 @@
+import IUser from '../components/interfaces/IUser';
+import axios from 'axios';
+
+const createHttpConfig = (token?: string) => axios.create({
+    baseURL: "http://api.safely-app.fr:8081",
+    headers: {
+        "Content-type": "application/json",
+        Authorization: `Bearer ${token}`
+    }
+});
+
+class Safeplace {
+    getAll(token: string) {
+        return createHttpConfig(token).get("/");
+    }
+
+    get(id: string, token: string) {
+        return createHttpConfig(token).get(`/safeplace/${id}`);
+    }
+
+    update(id: string, data: IUser, token: string) {
+        return createHttpConfig(token).put(`/safeplace/${id}`, data);
+    }
+
+    delete(id: string, token: string) {
```

Comment:
Same for the delete ? What are the criteria that could make us delete a Safeplace ?

Reply:
Some report from the community, some data or information not accurate. Or simply the end of the SafePlace Activity (eg a shop who close definitely)